sample-app/src/components/FilterSearch.tsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from "react";
+import { useAnswersActions, FilterSearchResponse, SearchParameterField } from '@yext/answers-headless-react';
+import InputDropdown from "./InputDropdown";
+import renderWithHighlighting from "./utils/renderWithHighlighting";
+import DropdownSection, { Option } from "./DropdownSection";
+import { processTranslation } from "./utils/processTranslation";
+import { useSynchronizedRequest } from "../hooks/useSynchronizedRequest";
+
+const SCREENREADER_INSTRUCTIONS = 'When autocomplete results are available, use up and down arrows to review and enter to select.'
+
+export interface FilterSearchProps {
+  title: string,
+  sectioned: boolean,
+  searchFields: Omit<SearchParameterField, 'fetchEntities'>[],
+  screenReaderInstructionsId: string,
+  cssClasses: {
```

Comment:
Let's make all these classes optional, in accordance with what Connor's been doing.

Reply:
I thought it was the customCssClasses that were optional? The cssClasses are currently required for `InputDropdown` and other components, so they would have to be required somewhere. Did you mean that I should move the classes back into FilterSearch to use as default css classes and then make this an optional `customCssClasses` prop?

Reply:
The `cssClasses` prop (which I'm calling `customCssClasses` ) and the classes themselves should be optional. In my search bar styling PR, I made the css classes of the `InputDropdown` optional, and we should update any other components so that they are optional as well

Reply:
Updated!